### ISSUE TITLE
Synchronization related

### ISSUE THREAD

**Issue by hope366** [yzyray/lizzieyzy#191]
Regarding "help_jp.docx" which is the manual (Japanese version) of the synchronization tool, it seems that the image display was strange, so I created a new one by imitating the Chinese version and English version.
[help_jp.docx](https://github.com/yzyray/lizzieyzy/files/9151574/help_jp.docx)

By the way, I have one question.
A lot of changes have been made to ReadBoard.java in c42630a. What exactly is this?
I downloaded the latest main branch and checked the contents of ReadBoard.java, but the changes were not reflected. Why is this?

**Comment by yzyray** [comment]
I cant open your file unless change the name from 'help_jp.docx' to 'help_jp.doc',and the screenshot looks pretty drak,is it your intend?
![image](https://user-images.githubusercontent.com/49384584/180343550-c7cd5e46-d525-442a-ad5f-4c1511357187.png)
![image](https://user-images.githubusercontent.com/49384584/180343589-e6d0f539-688f-468b-a19b-0fed1b1b1e66.png)

>A lot of changes have been made to ReadBoard.java in https://github.com/yzyray/lizzieyzy/commit/c42630addee1da8ef9a6849c93d6bb34dca81197. What exactly is this?

I only add this line: 
![image](https://user-images.githubusercontent.com/49384584/180343960-afd33631-8b82-4729-9b50-66b22e4e0cbf.png)

**Comment by hope366** [comment]
I created help_jp.docx using LibreOffice. In my environment I can open it as .docx.
When I tried to save in docx format, I got the following warning, but I forcibly saved it in docx format. This may be the reason why it looks dull in your environment.
>This document probably contains formats and content that cannot be saved with the currently selected file format “Word 2007–365”.

I uploaded a file saved in .odt format. How about this?
[help_jp.odt](https://github.com/yzyray/lizzieyzy/files/9166239/help_jp.odt)




>and the screenshot looks pretty drak,is it your intend?

It looks fine in my environment.
![無題](https://user-images.githubusercontent.com/63999713/180400177-b427664f-1149-4210-a5d8-20bc926836d3.png)
![1無題](https://user-images.githubusercontent.com/63999713/180400189-c1aa5590-401e-42e4-bbfa-50385107cb6b.png)

>I only add this line:

The file I uploaded last time lacked that added part, so I added a new one and re-uploaded it (.odt file above)
